src/aat/resources/features/F-047/S-572.td.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "must return negative response HTTP-400 when request contains a malformed jurisdiction ID",
+  "_guid_": "S-572",
+  "_extends_": "F-047_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "contains a malformed jurisdiction ID",
+    "code is HTTP-400"
+  ],
+  "request": {
+    "pathVariables": {
+      "uid": "dummyAuthentication"
```

Comment:
uid should be correct value.
Please update jid with a malformed value.
Also add this scenario in jira RDM-6387.

Reply:
done